src/components/Modals/Onboarding/OnboardingReview.vue

```diff
@@ -89,13 +89,16 @@
         <div class="onboarding-selectWrapper">
           <div class="onboarding-selectWrapper-reviewExam">
             <div class="alignLeft">
-              <label class="onboarding-label">AP Credits</label>
+              <label class="onboarding-label onboarding-label--header">AP Credits</label>
+              <label class="onboarding-label">Subject</label>
               <div v-for="(exam, index) in onboardingData.exam" :key="'AP' + index">
                 <label v-if="exam.type == 'AP'" class="onboarding-label--review">{{
                   exam.subject
                 }}</label>
               </div>
-              <label class="onboarding-label addSpaceTop">IB Credits</label>
+              <label class="onboarding-label addSpaceTop onboarding-label--header"
+                >IB Credits</label
+              >
```

Comment:
I think IB Credits should also have a label called Subject like AP Credits in the review page of Onboarding

Reply:
I don't follow

Reply:
There is no Subject for IB Credits as seen below:
![Screen Shot 2021-03-26 at 11 46 02 AM](https://user-images.githubusercontent.com/54298311/112657473-d945d300-8e28-11eb-994c-f754d37245d3.png)